chromium_src/chrome/browser/ui/views/bookmarks/bookmark_button.cc

```diff
@@ -12,5 +15,41 @@ void InstallNoHighlightPathGenerator(View* view) {
 }  // namespace views
 
 #define InstallPillHighlightPathGenerator InstallNoHighlightPathGenerator
+#define BookmarkButtonBase BookmarkButtonBase_ChromiumImpl
+#define BookmarkButton BookmarkButton_ChromiumImpl
 #include "src/chrome/browser/ui/views/bookmarks/bookmark_button.cc"
+#undef BookmarkButton
+#undef BookmarkButtonBase
 #undef InstallPillHighlightPathGenerator
```

Comment:
How about this? With this we don't need the header file override and others.

```suggestion
// Skips highlightPath the default highlight path is what we want.
// And in order to render label clearly over the ink drop, it should have its
// own layer. Otherwise, the ink drop will be rendered over the label.
#define InstallPillHighlightPathGenerator(view)       \
  Label* bookmark_label = label();                    \
  bookmark_label->SetPaintToLayer();                  \
  bookmark_label->SetSubpixelRenderingEnabled(false); \
  bookmark_label->layer()->SetFillsBoundsOpaquely(false);

#include "src/chrome/browser/ui/views/bookmarks/bookmark_button.cc"

#undef InstallPillHighlightPathGenerator

```

Reply:
Updated. good idea! :)